### PR TITLE
[Cosmos] update release versions for Cosmos SDKs

### PR DESCRIPTION
**Pull request by simorenoh**
September Release

azure-cosmos                              4.36.0
azure-spring-data-cosmos          3.27.0
azure-cosmos-encryption           1.6.0
azure-cosmos-spark_3-1_2-12    4.13.0
azure-cosmos-spark_3-2_2-12    4.13.0

### CODE REVIEW

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

File: sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
-### 3.27.0-beta.1 (Unreleased)
+### 3.27.0 (2022-09-15)
 
 #### Features Added
 * Exposed `maxBufferedItemCount` feature from `CosmosQueryRequestOptions` through `application.properties` flag - See [PR 30921](https://github.com/Azure/azure-sdk-for-java/pull/30921)
 
-#### Breaking Changes
-
 #### Bugs Fixed
 * Fixing pagination bug when performing a cross-partition query to fill every page and fix the total page count reporting. - See [PR 30694](https://github.com/Azure/azure-sdk-for-java/pull/30694)
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:10]: We are missing other changes section for spring-data-cosmos. Please add that. (see the previous release notes for reference)

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks @simorenoh

File: sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md (hunk 1 above)
Comment (simorenoh) [reply sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:10]: done!

File: sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,8 @@
...
 
+#### Other Changes
+* Updated `azure-cosmos` to version `4.36.0`.
 ### 3.26.0 (2022-08-19)
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:13]: nit: spacing.

**Review by kushagraThapar — APPROVED** [review]

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

**Pull request merged**